frontend/src/components/contacts/ContactDetailsContent.tsx

```diff
@@ -141,98 +191,168 @@ export default function ContactDetailsContent() {
-                </h3>
-              </div>
-              <p className="text-2xl font-bold">
-                ${summary?.totalSpent?.toLocaleString() || "0"}
-              </p>
-            </div>
-            <div className="bg-white rounded-lg shadow-sm p-4">
-              <div className="flex justify-between items-center mb-2">
-                <h3 className="text-sm font-medium text-gray-500">
-                  Total Transactions
-                </h3>
+              
+              <div className="grid grid-cols-1 gap-3 md:grid-cols-3">
+                <div className="flex items-center gap-2 text-gray-600">
+                  <MapPin className="h-4 w-4 text-gray-400" />
+                  <span>
+                    {contact.city || "Boston"}, {contact.state || "MA"}
+                  </span>
+                </div>
+                <div className="flex items-center gap-2 text-gray-600">
+                  <Mail className="h-4 w-4 text-gray-400" />
+                  <span>{contact.email}</span>
+                </div>
+                <div className="flex items-center gap-2 text-gray-600">
+                  <Phone className="h-4 w-4 text-gray-400" />
+                  <span>{contact.phone || "999-999-9999"}</span>
+                </div>
               </div>
-              <p className="text-2xl font-bold">
-                {summary.totalTransactions} Transactions
-              </p>
             </div>
-            <div className="bg-white rounded-lg shadow-sm p-4">
-              <div className="flex justify-between items-center mb-2">
-                <h3 className="text-sm font-medium text-gray-500">
-                  Total Emissions
-                </h3>
-              </div>
-              <p className="text-2xl font-bold">
-                {summary.totalEmissions} Tn CO₂
-              </p>
+            
+            <div className="mt-4 md:mt-0">
+              <Button 
+                variant="ghost" 
+                className="flex items-center text-blue-600 hover:text-blue-800"
+                onClick={() => console.log("Edit Contact")}
+              >
+                Edit Contact
+              </Button>
             </div>
           </div>
+        </div>
+
+        <div className="grid grid-cols-1 md:grid-cols-7 gap-6">
+          {/* Left side - Transactions */}
+          <div className="md:col-span-5">
+            <h2 className="text-xl font-bold mb-4">All Transactions</h2>
+            <div className="bg-white rounded-lg shadow-sm overflow-hidden">
+              <Tabs defaultValue="transactions" className="w-full">
+                <div className="border-b">
+                  <TabsList className="p-0 bg-transparent w-full flex rounded-none border-b">
+                    <TabsTrigger 
+                      value="transactions" 
+                      className="data-[state=active]:border-b-2 data-[state=active]:border-blue-600 data-[state=active]:shadow-none rounded-none flex-1"
```

Comment:
```suggestion
                      className="data-[state=active]:border-b-2 data-[state=active]:border-freshSage data-[state=active]:shadow-none rounded-none flex-1"
```

I pulled in our figma colors and think green over blue could be good here (and in the other TabsTriggers as well)